### PR TITLE
Add configurations table to README.md

### PR DESCRIPTION
**Pull request by akulakovaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,21 +112,41 @@ Email: john@barong.io, password: Am8icnzEI3d!
...
+--- | --- | ---
+`app.name` | global application name | `"OpenDax"`
+`app.domain` | base domain name to be used | `app.local`
+`subdomain` | subdomain of your application | `www`
+`render_protect` | enable read-only mode for rendered files | `false`
+`csrfEnabled` | enable csrf token on barong | `false`
+`ssl.enabled` | enable SSL certificate generation | `false`
+`ssl.email` | email address to use for SSL generation requests | `"support@example.com"`
+`images` | application images tags
+`vendor.frontend` | vendor is a tool to develop front with opendax, put there a link to you front repo | `git@github.com:openware/baseapp.git`
```
Comment (dpatsora) [inline README.md]: I think `develop frontend application` would be better

File: README.md [hunk 2]
```diff
@@ -112,21 +112,41 @@ Email: john@barong.io, password: Am8icnzEI3d!
...
+`vault.token` | root token to unseal the vault | `changeme `
```
Comment (dpatsora) [inline README.md]: We don't use root token, to unseal vault.
Just put "Vault root token"

Comment (vshatravenko) [reply README.md]: Let's put Vault auth token, it doesn't have to be root :)

File: README.md (hunk 1 above)
Comment (vshatravenko) [reply README.md]: Vendor is not a tool, it's just a way of organizing your code in this case

**Pull request merged**